inc/namespace.php

```diff
@@ -176,6 +185,7 @@ function enqueue_scripts() {
 						'CognitoRegion' => defined( 'ALTIS_ANALYTICS_COGNITO_REGION' ) ? ALTIS_ANALYTICS_COGNITO_REGION : null,
 						'CognitoEndpoint' => defined( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT' ) ? ALTIS_ANALYTICS_COGNITO_ENDPOINT : null,
 					],
+					'Noop' => (bool) apply_filters( 'altis.analytics.preview', false ),
```

Comment:
Lets give this filter a more generic name - the preview is just one context we've found where we don't want to record events so let's call it `altis.analytics.noop`.

We should document the filter on the readme too.